kong/plugins/consumer-rate-limiting/handler.lua

```diff
@@ -17,7 +17,7 @@ function ConsumerRateLimiting:access(conf)
   local user = ngx.ctx.authenticated_consumer
   local api = ngx.ctx.api
 
-  local ctrl = Controller(gateway.CounterGateway(), gateway.QuotaGateway(), timeutils.PeriodGenerator())
+  local ctrl = Controller(gateway.CounterGateway(), gateway.QuotaGateway(), timeutils.PeriodGenerator("months"))
```

Comment:
The target for #2 is to create a additional configuration option (can be called `period` or `cadence`) to the plugin, so the user can define it, when enabling the plugin. Hardcoding it doesn't extend the functionality of the plugin.